test/token-paymaster/biconomy-token-paymaster-specs.ts

```diff
@@ -206,7 +206,7 @@ describe("Biconomy Token Paymaster", function () {
       );
 
       expect(res.priceSource).to.equal(1);
-      expect(res.fee).to.equal(ethers.constants.Zero);
+      expect(res.priceMarkup).to.equal(1100000);
```

Comment:
Use the DEFAULT_FEE_MARKUP variable instead of hardcode value 1100000

Reply:
fixed